tests/Maker/FunctionalTest.php

```diff
@@ -387,6 +395,94 @@ function (string $output, string $directory) {
                 ),
         ];
 
+        $makerTestAuthenticatorLoginFormUserEntity = MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeAuthenticator::class),
+            [
+                // authenticator type => login-form
+                1,
+                // class name
+                'AppCustomAuthenticator',
+                // controller name
+                'SecurityController',
+            ]
+        )
+            ->addExtraDependencies('doctrine')
+            ->addExtraDependencies('security')
+            ->addExtraDependencies('twig')
```

Comment:
This extra dependency thing should be rare. `doctrine` make sense to me - that's not a requirement of the Maker, but we're testing that case. But, shouldn't security be a dependency? I suppose `twig` is correct to stay here too, because it's only a conditional dependency - i.e. if the choose the "login form" version. Our test system doesn't support those "optional" dependencies. But, we *should* do a manual check in the maker to make sure they have this if the choose the login-form option. I think I do this in MakeUser - I do a manual check for Doctrine, if they choose the entity option.